config/routes.rb

```diff
@@ -8,5 +8,7 @@
     resources :companies
     resources :pages, only: %i[show index]
     resources :tenants
+    resources :dashboard, only: [:index]
+    get '/', to: 'dashboard#index'
```

Comment:
use `root to: 'dashboard#index'` instead

Reply:
Yes, it's much elegant, done.